CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ This also works just as well with [Visual Studio Code's Remote Development](http
 
 
 ## Building the encore command from source
-To build from source, [build the dashboard](#Building-the-Development-Dashboard) and simply run `go build ./cli/cmd/encore`.
+To build from source, [build the dashboard](#Building-the-Development-Dashboard) and simply run `go build ./cli/cmd/encore` and `go build ./cli/cmd/git-remote-encore`.
```

Comment:
`git-remote-encore` doesn't really require ongoing development so it might be better to suggest to `go install` it, so it ends up in `$PATH` automatically

Reply:
yes my bad 🙇🏽